tidb-cloud/dev-guide-bi-looker-studio.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Connect to TiDB Serverless with Looker Studio
+summary: Learn how to connect to TiDB Serverless using Looker Studio.
+---
+
+# Connect to TiDB Serverless with Looker Studio
+
+TiDB is a MySQL-compatible database, TiDB Serverless is a fully managed TiDB offering, and [Looker Studio](https://lookerstudio.google.com/) is a free web-based BI tool that can visualize data from various sources.
+
+In this tutorial, you can learn how to connect to your TiDB Serverless cluster with Looker Studio.
+
+> **Note:**
+>
+> Most steps in this tutorial work with TiDB Dedicated as well. However, for TiDB Dedicated, you need to import your dataset following [Import data from files to TiDB Cloud](/tidb-cloud/tidb-cloud-migration-overview.md#import-data-from-files-to-tidb-cloud) and get the connection information for your cluster following [Connect to TiDB Dedicated](/tidb-cloud/connect-via-standard-connection.md). In addition, when connecting to TiDB Dedicated, you need to allow access from `142.251.74.0/23`. For more information about connection from Looker Studio, see [Looker Studio documentation](https://support.google.com/looker-studio/answer/7088031#zippy=%2Cin-this-article).
+
+## Prerequisites
+
+To complete this tutorial, you need:
+
+- A Google account
+- A TiDB Serverless cluster
+
+**If you don't have a TiDB Serverless cluster, you can create one as follows:**
+
+- [Create a TiDB Serverless cluster](/develop/dev-guide-build-cluster-in-cloud.md)
+
+## Step 1. Import a dataset
+
+You can import the S&P 500 dataset provided in the interactive tutorial of TiDB Serverless.
+
+1. Navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page, and click **?** in the lower-right corner. A **Help** dialog is displayed.
+
+2. In the dialog, click **Interactive Tutorials**, and then click **S&P 500 Analysis**.
+
+3. Select your TiDB Serverless cluster, and then click **Import Dataset** to import the S&P 500 dataset to your cluster.
```

Comment:
As "This usually takes 60 seconds" is already provided on the GUI, which is easier for users to see, perhaps, it's better not to include info here in the doc in case the duration changes in the future.